### PR TITLE
fix: nodes list stops working on sort by uptime scroll down

### PR DESCRIPTION
**Pull request by astandrik**
Closes https://github.com/ydb-platform/ydb-embedded-ui/issues/1383

[Stand](http://ydb-vla-dev02-005.search.yandex.net:8765/monitoring/tenant?tenantPage=diagnostics&diagnosticsTab=nodes&topQueriesDateFrom=1721228339103&topQueriesDateTo=1721314739103&selectedConsumer=&name=%2Fdev02)

### Test Status: <span style="color: green;">✅ PASSED</span>
📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1424/)

| Total | Passed | Failed | Flaky | Skipped |
|:-----:|:------:|:------:|:-----:|:-------:|
| 128 | 128 | 0 | 0 | 0 |

### Bundle Size: 🔽
Current: 78.96 MB | Main: 79.14 MB
Diff: 0.17 MB (-0.22%)

✅ Bundle size decreased.

<details>
<summary>ℹ️ CI Information</summary>

- Test recordings for failed tests are available in the full report.
- Bundle size is measured for the entire 'dist' directory.
- 📊 indicates links to detailed reports.
- 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
</details>

### CODE REVIEW

**Comment by astandrik** [comment]
tests fail because of long table mocks - this is an expected state

File: src/components/PaginatedTable/useScrollBasedChunks.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        const visibleStartIndex = Math.floor(scrollTop / itemHeight);
+        const visibleEndIndex = Math.min(
+            Math.ceil((scrollTop + clientHeight) / itemHeight),
+            totalItems - 1,
+        );
+
+        const startChunk = Math.floor(visibleStartIndex / chunkSize);
+        const endChunk = Math.floor(visibleEndIndex / chunkSize);
+
+        const newActiveChunks = getArray(endChunk - startChunk + 1).map(
```
Comment (astandrik) [inline src/components/PaginatedTable/useScrollBasedChunks.ts:40]: let load 1 more chunk before and 1 more chunk after the view area

Comment (astandrik) [reply src/components/PaginatedTable/useScrollBasedChunks.ts:40]: sure, great idea!

Comment (astandrik) [reply src/components/PaginatedTable/useScrollBasedChunks.ts:40]: After discussion with @adameat decided to load +1 chunk ahead

**Review by artemmufazalov — DISMISSED** [review]

**Comment by artemmufazalov** [comment]
Please, remove all code for test

**Pull request merged**